config/config.template.ini

```diff
@@ -114,6 +114,19 @@ ricd = Unwetter
 # Usually sent periodically, separated by comma
 netIdent_ric = 0174760, 1398098
 
+[ExpressAlarm]
+# Using Express-Alarm (0 - off | 1 - on)
+expressAlarm = 0
+
+# time limit for alarms that do not belong to the Express-Alarm sequence

```

Comment:
time limit - in Minuten? Sekunden? Bitte noch vermerken...

Reply:
Da hast du natürlich Recht. Was bringt die Option ohne die Einheit zu wissen.